cmd/wallet-web/src/pages/Vaults.vue

```diff
@@ -54,30 +50,41 @@ export default {
   },
   data() {
     return {
-      credentialsFound: '',
+      numofCreds: 0,
       vaults: [],
     };
   },
-  created: function () {
+  created: async function () {
     const { user, token } = this.getCurrentUser().profile;
     this.username = this.getCurrentUser().username;
     const credentialManager = new CredentialManager({ agent: this.getAgentInstance(), user });
-    this.fetchAllCredentials(credentialManager.getAll(token));
+    // Fetch all the credentials stored.
```

Comment:
as discussed please put TODO here as well, so that total count will be count of credentials from all the vaults.

It means, no credentials to be saved without vault ID.

Reply:
done